test/jsdom/misc.js

```diff
@@ -115,7 +115,7 @@ describe("jsdom/miscellaneous", () => {
       process.nextTick(() => {
         req.emit("response", res);
         process.nextTick(() => {
-          res.emit("data", "window.attachedHere = 123");
+          res.emit("data", new Buffer("window.attachedHere = 123"));
```

Comment:
Oh, I wasn't looking at the larger context for this code. It makes sense. Thanks.